account_invoice_line_sequence/views/account_invoice_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']" position="before">
+                <field name="max_line_sequence" invisible="1" />
+            </xpath>
+            <xpath expr="//field[@name='invoice_line_ids']" position="attributes">
+                <attribute
+                    name="context"
+                    operation="python_dict"
+                    key="default_sequence"
+                >
```

Comment:
Found the issue, although not yet solved it.

This piece invokes code of `base_view_inheritance_extension`, but there is a bug: Odoo's [context.get('default_type')](https://github.com/OCA/OCB/blob/13.0/addons/account/views/account_move_views.xml#L674) is mistakenly seen as being [not a variable](https://github.com/OCA/server-tools/blob/13.0/base_view_inheritance_extension/models/ir_ui_view.py#L84) and as such is quoted.

Solution angles include:

- Patching `is_variable` function
- Backporting https://github.com/OCA/server-tools/pull/2216 and https://github.com/OCA/server-tools/pull/2305/files that reimplement this
- Doing this part without needing base_view_inheritance_extension (tips anyone?)

Reply:
I chose the second option and backported to https://github.com/OCA/server-tools/pull/2306

Reply:
Hi @thomaspaulb , Thanks for the fix, I agree option 2 is the better one.